cristin-person/src/main/java/no/unit/nva/cristin/person/model/nva/Person.java

```diff
@@ -0,0 +1,191 @@
+package no.unit.nva.cristin.person.model.nva;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+import static no.unit.nva.cristin.person.Constants.PERSON_CONTEXT;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import java.net.URI;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import nva.commons.core.JacocoGenerated;
+
+@JacocoGenerated
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+@JsonPropertyOrder({"@context"})
+public class Person {
+
+    @JsonInclude(NON_NULL)
+    @JsonProperty("@context")
```

Comment:
Minor comment: in case of static fields, I would suggest putting the `@JsonProperty` at the getter and the setter making it empty. We avoid thus to have a field that is only a copy of a static value. It is also more clear about our intention.